incubator/mariadb-cluster/templates/slave-configmap.yaml

```diff
@@ -0,0 +1,16 @@
+{{- if .Values.replication.enabled }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "slave.fullname" . }}
+  labels:
+    app: {{ template "slave.fullname" . }}
```

Comment:
`app` label should be `app: "{{ template "name" . }}"`

Reply:
done